test/property_test.c

```diff
@@ -145,6 +145,52 @@ static int test_property_query_value_create(void)
     return r;
 }
 
+static const struct {
+    int query;
+    const char *ps;
+} parse_error_tests[] = {
+    { 0, "n=1, n=1" },          /* duplicate name */
```

Comment:
Will Overriding global property properties will still work with this change?

Reply:
Yes.

They are merged as property lists not concatenated as strings.